cpp/src/sampling/detail/conversion_utilities.cu

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "sampling/detail/sampling_utils.hpp"
+
+#include <cugraph/utilities/misc_utils.cuh>
+
+namespace cugraph {
+namespace detail {
+
+rmm::device_uvector<int32_t> convert_starting_vertex_offsets_to_labels(
```

Comment:
`starting_vertex_offsets`=>`starting_vertex_label_offsets`?

Reply:
And should we really define this function and create one additional layer of indirection? What this function does is just calling `expand_sparse_offsets` so why not just directly call `expand_sparse_offsets`?

Reply:
Right we can directly call expand_sparse_offsets but I believe an additional layer of indirection was added for a better description of the operation being performed which is converting the starting_vertex_label_offsets to labels. And this is done through the expand_sparse_offsets method. @ChuckHastings any comment on this?